### DISCUSSION TITLE
`colima start` triggers a container to auto run

### DISCUSSION THREAD

**Discussion by vbhavsar** [abiosoft/colima#793] [Q&A] [answered]
Every time I start colima with `colima start`, the container `bitnami/zookeeper:3.8` automatically starts to run. How do I stop it from auto running? I found nothing obvious in documentation or colima config.

**Comment by gabyx** [comment, marked as answer]
Isnt this a docker thing. That it will restart the container when it was launched in detached mode?

**Reply by vbhavsar** [reply]
Yes, it turned out to be because the container was started in detached mode with `--restart=always` flag. I updated the restart config via
```
docker update --restart no <container_id>
```
and that fixed it. Thanks for the pointer!
